docs/data/date-pickers/validation/validation.md

```diff
@@ -1,6 +1,6 @@
 ---
 productId: x-date-pickers
-components: DatePicker, DesktopDatePicker, MobileDatePicker, StaticDatePicker, TimePicker, DesktopTimePicker, MobileTimePicker, StaticTimePicker, DateTimePicker, DesktopDateTimePicker, MobileDateTimePicker, StaticDateTimePicker, DateRangePicker, DesktopDateRangePicker, MobileDateRangePicker, StaticDateRangePicker
+components: DatePicker, DesktopDatePicker, MobileDatePicker, StaticDatePicker, TimePicker, DesktopTimePicker, MobileTimePicker, StaticTimePicker, DateTimePicker, DesktopDateTimePicker, MobileDateTimePicker, StaticDateTimePicker, DateRangePicker, DesktopDateRangePicker, MobileDateRangePicker, StaticDateRangePicker, DateCalendar, DateRangeCalendar
```

Comment:
I can't see any demo using `DateRangeCalendar` on this page. 🤔 
Also, if we are adding the `DateCalendar` to the list of components, then it probably makes sense to also add the `TimeClock` as those are the two components that are listed plainly with validation toggled on.
Otherwise, as discussed, we can leave this different refactoring topic for a separate effort to eliminate the conflict of interest and any possible misalignments. 🙈 
```suggestion
components: DatePicker, DesktopDatePicker, MobileDatePicker, StaticDatePicker, TimePicker, DesktopTimePicker, MobileTimePicker, StaticTimePicker, DateTimePicker, DesktopDateTimePicker, MobileDateTimePicker, StaticDateTimePicker, DateRangePicker, DesktopDateRangePicker, MobileDateRangePicker, StaticDateRangePicker, DateCalendar
```